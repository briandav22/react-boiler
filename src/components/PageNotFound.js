import React from 'react';

const PageNotFound = ()=>(
    <div> This is from help dash </div>
);

export default PageNotFound;