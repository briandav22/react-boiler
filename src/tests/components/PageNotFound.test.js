import React from 'react';
import {shallow} from 'enzyme';
import PageNotFound from '../../components/PageNotFound';



test('Should render Page Note Found', ()=>{
    const wrapper = shallow(<PageNotFound/>)
    expect(wrapper).toMatchSnapshot()
});