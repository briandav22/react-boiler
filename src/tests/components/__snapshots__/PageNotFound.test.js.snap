// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Page Note Found 1`] = `
<div>
   404! - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
